app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -5,8 +5,9 @@
 // helper functions for views with a profile image. Meant to be mixed into views.
 
 define([
-  'models/profile-image'
-], function (ProfileImage) {
+  'models/profile-image',
+  'lib/profile-errors'
```

Comment:
Can you swap the order of these? I'm going to open a PR one of these days to alphabetize the dependencies, unless your webpack PR lands first.